src/registry_schemas/schemas/amalgamation.json

```diff
@@ -0,0 +1,105 @@
+                    "items": {
+                        "oneOf": [
+                            {
+                                "required": [
+                                    "role",
+                                    "identifier"
+                                ]
+                            },
+                            {
+                                "required": [
+                                    "role",
+                                    "foreignJurisdiction",
+                                    "legalName",
+                                    "corpNumber"
+                                ]
+                            }
+                        ],
+                        "properties": {
+                            "role": {
+                                "type": "string",
+                                "enum": [
+                                    "holding",
+                                    "amalgamating"
+                                ]
+                            },
+                            "identifier": { "type": "string" },
+                            "foreignJurisdiction": { 
+                                "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/foreign_jurisdiction" 
+                            },
+                            "legalName": { "type": "string" },
+                            "corpNumber": { "type": "string" }
+                        }
+                    }
+                },
+                "nameRequest": {
+                    "title": "The resulting business name and type",
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request"
+                },
+                "offices": {
+                    "registeredOffice": {
+                        "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+                    },
+                    "recordsOffice": {
+                        "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+                    },
+                    "required": [
+                        "registeredOffice"
+                    ]
+                },
+                "contactPoint": {
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/contactPoint"
+                },
+                "parties": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/parties#/definitions/party"
+                    }
+                },
+                "shareStructure": {
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/share_structure"
+                },
+                "courtOrder": {
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/court_order#/properties/courtOrder"
+                },
+                "courtApproval": {
+                    "title": "Amalgamation statement",
+                    "type": "boolean"
+                }
```

Comment:
courtApproval should allow null value too

Reply:
@severinbeauvais the FE wants to be able to store a null for drafts for `courtApproval` right?

Reply:
In a draft amalg filing, yes, this value could be null (or the property doesn't exist).

In the filed JSON, this property must be present and either be True or False (never null).

Reply:
ok, ignore what I said then @chenhongjing.  

We can remove the null option as null won't be there when we validate.